tests/test_time_series_graph.py

```diff
@@ -493,6 +493,17 @@ def test_from_adjacency_matrices(self):
         with self.assertRaises(AssertionError):
             _ = TimeSeriesCausalGraph.from_adjacency_matrices(matrices, variables)
 
+        # test for floating nodes
```

Comment:
What happens if floating node at ad different time lag? Should confirm they are not added if that is the intended behaviour.

Reply:
Already discussed offline. We can resolve this.